massa-node/base_config/config.toml

```diff
@@ -156,6 +156,10 @@
     # Max number of in connection overflowed managed by the handshake
     # that send a list of peers
     max_in_connection_overflow = 100
+    # Read limitation for a connection in bit by 0.100 milliseconds
+    max_bit_read = 40_000_000
+    # Write limitation for a connection in bit by 0.100 milliseconds
+    max_bit_write = 72_000_000
```

Comment:
The difference between the read and write are unexepected.
In P2P networks we expect upload to be equivalent to download, unless the node is special (eg. default bootstrap server).

Reply:
We can look together between Monday and Friday the graphic but it looked like that.

Reply:
Okay let's do that